setup.cfg

```diff
@@ -19,6 +19,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
+    pandas==1.5.3
```

Comment:
Any reason for not using the latest version?

Reply:
I was just using the same one for data2rdf since we are currently installing data2rdf and the sdk in parallel in the knowledge service. This might overcome conflicting versions.

Reply:
I suppose it would be better to upgrade pandas in the data2rdf as well

Reply:
We might need to test for `pandas>=2,<3` for both packages. And in the knowledge service.